plasmapy/atomic/particle_input.py

```diff
@@ -173,6 +176,25 @@ def particle_input(wrapped_function: Callable = None,
         def decorated_function(particle: Particle):
             return particle
 
+    This decorator may also be used to accept arguments using tuple
+    annotation containing specific number of elements or using list
+    annotation which accepts any number of elements in an iterable.
+    Returns a tuple of `~plasmapy.atomic.Particle`:
+
+    .. code-block:: python
+
+        from plasmapy.atomic import particle_input, Particle
+        @particle_input
+        def decorated_tuple_function(particles: (Particle, Particle)):
+            return particles
+        sample_particles = decorated_tuple_function(('He', 'Li'))
+
+        @particle_input
+        def decorated_list_function(particles: [Particle]):
+            return particles
+        sample_particles = decorated_list_function(('Al 3+', 'C'))
+        sample_particles = decorated_list_function(['He', 'Ne', 'Ar'])
```

Comment:
Yep, `(Particle, Particle)` accepts only 2 elements and `(Particle, Particle, Particle)` accepts 3 elements and so on.. (either as tuple or list).

while `[Particle]` accepts any number of `Particle` elements (either as tuple or list).

Rust programming language has a similar concept of passing vectors (think: lists in Python) and tuples of specific type to a function. I imagined it would be nice here as well. :smile: